tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -219,131 +220,151 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 			Expect(fw.AsKubeAdmin.HasController.WaitForComponentPipelineToBeFinished(component2, "", 2)).To(Succeed())
 		})
 
-		It("verifies that a release PipelineRun for each Component is created in managed namespace.", func() {
-			Eventually(func() bool {
-				prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
-				if err != nil || prList == nil || len(prList.Items) < 1 {
-					GinkgoWriter.Printf("Error getting Release PipelineRun:\n %s", err)
-					return false
+		It("tests that Snapshot is created for each Component", func() {
```

Comment:
As you can see below, `snapshot1` and `snapshot2` vars are used for getting Release CRs (which are then used for getting Release PipelineRuns). 
Afaik Snapshot CR is also a prerequisite for creation of Release CR, so I decided to add the assertion for creation of Snapshot CR here in case something went wrong between the phase when Component PipelineRun finishes and Release CR is created.

But if you think it'd be better to remove this test spec, I'm happy to move "getting the snaphots logic" below into the
```
It("tests that associated Release CR is created for each Component's Snapshot", func() {
```
spec

Reply:
I see, 
Let's keep it as is for now , if anything will be needed I will update. 
thanks